### ISSUE TITLE
Accessibility issues: Feature page should be selected when the feature selection is changed

### ISSUE THREAD

**Issue by jhkimnew** [Microsoft/IIS.WebManager#349]
When navigating features, we had to hit Enter key to select the feature.
I compared Registry of WAC, which selects items automatically when changing the registry key path.
So, we decided to do the same thing for IIS WAC extension.

**Comment by jhkimnew** [comment]
The same problem exists on selecting tab.
